src/Jobs/Jobs.jsx

```diff
@@ -9,7 +9,7 @@ import jobsList from './JobsList';
 import JobCard from './JobCard';
 
 const Jobs = () => (
-	<div>
+	<React.Fragment>
```

Comment:
Is it not common practice to import with destructuring so that Fragment can be used instead of React.Fragment?

Reply:
And why do you need a Fragment if there is only one child node?